tests/slo/backup_quality/test_quality_query_specification.py

```diff
@@ -0,0 +1,16 @@
+from unittest import TestCase
+
+from src.slo.backup_quality.quality_query_specification import QualityQuerySpecification
+from src.commons.config.configuration import Configuration
+from mock import patch, PropertyMock
+
+
+class TestQualityQuerySpecification(TestCase):
+    @patch.object(Configuration, 'projects_to_skip', return_value=['BBQ-project-id', 'ta-testing-project', '123'],
```

Comment:
As this is open source project could you rename ta-testing-project to sth else?

Reply:
done